src/ngx_http_lua_worker.c

```diff
@@ -99,6 +99,17 @@ ngx_http_lua_ffi_worker_pid(void)
 }
 
 
+int
+ngx_http_lua_ffi_master_pid(void)
+{
+#if (nginx_version >= 1013008)
+    return (int) ngx_parent;
```

Comment:
Wen `master_process off;` is configured inside `nginx.conf`, then this will yield a wrong value (a different value than the one in the `nginx.pid` file). In this case, we should return `ngx_pid` instead.